Source/Core/Common/MathUtil.h

```diff
@@ -117,19 +117,21 @@ double ApproximateReciprocal(double val);
 template<class T>
 struct Rectangle
 {
-	T left;
-	T top;
-	T right;
-	T bottom;
+	T left{};
```

Comment:
Yep, if an arithmetic type is given (currently we don't static assert against that)